build.js

```diff
@@ -32,7 +32,7 @@ const pkgOptions = {
     nodePaths,
     outbase: './pkg',
     outdir: "./dist",
-    target: ['es2020'],
+    target: ['es2021'],
```

Comment:
This should be lib, we don't want to target es2021 in the end result of the transpilation https://github.com/cockpit-project/cockpit/pull/21276/commits/5f68dfee226e155feb241ebe2cf1757e55cac996

Reply:
Yes, but I don't want half of our code 2021 and half 2020. Let's stick to *one* standard.

Reply:
Alright! Would have also liked a .eslintrc.json downgrade but we can do that later.